open-subshare/REPWRITERSPECS/BANNO.NEWSUBCREATE.V1.POW

```diff
@@ -1028,7 +1073,7 @@ PROCEDURE SHAREOUTPUT
          IF J>0 THEN
           DO
            IF SACSHARETYPES(I,J-1)<>"" OR
-              SACSHARETYPEEXISTS(J-1)=TRUE THEN
+              SACSHARETYPEEXISTS(J-1)<>TRUE THEN
```

Comment:
If I'm interpreting the code correctly, The SACSHARETYPEEXISTS flag is set when a test is done (lines 997-1001) to make sure the share types listed are defined.  These two lines of code are essentially looking to see that the previous one (J-1) is not blank and exists (is valid). Therefore, I'm not sure why the test to make sure that it would not be equal to true.... meaning that it doesn't exist. Is this being changed to "<>TRUE" here because, in effect, by removing - or in this case reversing - the test, we're correcting for an error in the code somewhere else? Would we get the same results by just using the "SACSHARETYPES(I,J-1)<>""" portion of the test?